src/auth.js

```diff
@@ -103,7 +104,9 @@ exports.configure = ({
   })
 
   passport.deserializeUser(async function (id, next) {
-    const user = await Users.findOne({id})
+    const user = createUser(
+      await Users.findOne({id})
```

Comment:
will probably result in `Uncaught TypeError: Cannot read property 'roles' of undefined` and the below `!user` check will never run